src/components/imagecard/imagecard.jsx

```diff
@@ -0,0 +1,17 @@
+import '../../pages/home-page/home-page.css'
+import { NavLink } from 'react-router-dom';
+
+const Imagecard=({ category }) =>{
+  const cakecategory=category.category;
+  const { categoryName, image, alt } = cakecategory;
```

Comment:
You can restructure all in single line

``` const {categoryName, image, alt}=category?.category```
Be sure to check for undefined values during initial render